code/ui/lisibilite_ui_app/views.py

```diff
@@ -0,0 +1,119 @@
+from django.shortcuts import render, redirect
```

Comment:
May be for next PR. There are three views in this file. Would prefer splitting it

Reply:
Sure. Something to ponder.